synapse/storage/data_stores/main/directory.py

```diff
@@ -159,10 +160,28 @@ def _delete_room_alias_txn(self, txn, room_alias):
 
         return room_id
 
-    def update_aliases_for_room(self, old_room_id, new_room_id, creator):
+    def update_aliases_for_room(
+        self, old_room_id: str, new_room_id: str, creator: Optional[str] = None,
+    ):
+        """Repoint the aliases from one room to another
+
+        Args:
+            old_room_id:
+            new_room_id:
+            creator: The creator of the alias mapping. If None, don't update
```

Comment:
this is unclear: I'm not sure if I should be setting it to the previous creator of the old alias, or the new one. And where it says "don't update", I initially read that as meaning that the alias wouldn't be updated at all.
```suggestion
            creator: The user to record as the creator of the new mapping. 
                If None, the creator will be left unchanged.
```